### PR TITLE
Undo last points method to the draw interraction

### PR DESCRIPTION
**Pull request by geonux**
The pull request adds the ability to delete the last segments added during editing with the draw interraction.

Usage sample  :

```

var draw_interaction = new ol.interaction.Draw( {
  type : "Polygon"
});

// Remove last point on "delete" key when drawing a feature.
var onKeyboardEvent = function(event) {
  if (draw_interaction && event.keyCode === 8) {
    draw_interaction.removeLastPoint();
    event.preventDefault();
  }
}

$document.bind('keydown', onKeyboardEvent);

### CODE REVIEW

File: src/ol/interaction/drawinteraction.js [hunk 1]
```diff
@@ -618,6 +618,35 @@ ol.interaction.Draw.prototype.addToDrawing_ = function(event) {
 
 
 /**
+ * Remove last drawed point of the currently edited feature.
```
Comment (probins) [inline src/ol/interaction/drawinteraction.js]: I'd say 'Remove last point of the feature currently being drawn.'
Or perhaps 'Remove last drawn point.' or 'Cancel last point drawn.'

**Comment by geonux** [comment]
The proposed method tries to respond to issues posted via the links below, works on lines and polygons, and directly updates the preview (do not needs mouse move on the map to refresh).

http://gis.stackexchange.com/questions/152119/undo-last-point-when-drawing-linestring-possible-regression
http://stackoverflow.com/questions/28968064/remove-last-point-from-draw-interaction-in-ol3
https://groups.google.com/forum/#!msg/ol3-dev/O60t4yvJRh4/DAh7KS1Bp4sJ

**Comment by ahocevar** [comment]
Looks good @geonux. If you haven't already, please submit a [CLA](https://github.com/openlayers/CLA) so we can merge this.

**Comment by geonux** [comment]
CCLA in progress...

File: src/ol/interaction/drawinteraction.js [hunk 2]
```diff
@@ -618,6 +618,35 @@ ol.interaction.Draw.prototype.addToDrawing_ = function(event) {
...
+    coordinates = this.sketchCoords_[0];
+    coordinates.splice(-2, 1);
+    sketchLineGeom = this.sketchLine_.getGeometry();
+    goog.asserts.assertInstanceof(sketchLineGeom, ol.geom.LineString,
+        'sketchLineGeom must be an ol.geom.LineString');
+    sketchLineGeom.setCoordinates(coordinates);
+    this.geometryFunction_(this.sketchCoords_, geometry);
+  }
+
+  if (coordinates.length === 0) this.finishCoordinate_ = null;
```
Comment (tschaub) [inline src/ol/interaction/drawinteraction.js]: Please update this to separate conditional statements with a curly brace (per [the style guide](https://github.com/openlayers/ol3/blob/master/CONTRIBUTING.md#follow-openlayers-3s-coding-style)).

**Comment by geonux** [comment]
The CCLA was mailed today.

**Comment by ahocevar** [comment]
Thanks @geonux.

**Pull request merged**